src/ser.rs

```diff
@@ -151,6 +153,12 @@ impl<T> ToStfu8 for T
 where
     T: Borrow<PathBuf>,
 {
+    #[cfg(target_os = "wasi")]
```

Comment:
this is identical to unix. I think you can use `cfg(any(...))` for such a case, no?

Reply:
You're right! I didn't know about `cfg(any())`. I'll quickly change this.

Reply:
You're right! I wasn't aware of `cfg(any())`.